cloudflare2.php

```diff
@@ -14,7 +16,8 @@
 ?>
 <div id="root" class="cloudflare-partners site-wrapper"></div>
 <script>
-var absoluteUrlBase = '<?=plugins_url('/cloudflare/');?>'
+var absoluteUrlBase = '<?=plugins_url('/cloudflare/');?>';
+cfCSRFToken = '<?=CF\SecurityUtil::csrfTokenGenerate($wordpressAPI->getHostAPIKey(), $wordpressAPI->getUserId());?>';
```

Comment:
`getHostAPIKey()` returns null cause this plugin doesn't use the host API.  I don't know what we're going to use for the secret yet but need to log this as a TODO to follow up on.